### DISCUSSION TITLE
When does Tangaroa start?

### DISCUSSION THREAD

**Discussion by kumkee** [Te-Maramataka/mataripy#9] [Ideas]
In [this discussion](https://github.com/orgs/Te-Maramataka/discussions/4), there are some confusion on how to determine the starting night of Tangaroa.
There are two reliable sources I can find to convince you it is the 23rd night of the Māori lunar month.
### [Te Ara](https://teara.govt.nz/en/maramataka-the-lunar-calendar/page-2)
All 30 nights are listed there. The Tangaroa nights are _Tangaroa-ā-mua_, _Tangaroa-ā-roto_, and _Tangaroa-kiokio_. They are the 23rd to 25th in the list.
### [The Matariki Dates document](https://www.mbie.govt.nz/assets/matariki-dates-2022-to-2052-matariki-advisory-group.pdf)
The last 8 nights are listed in the official document, which don't 100% agree with the one in [te Ara](https://teara.govt.nz/en/maramataka-the-lunar-calendar/page-2). However, _Tangaroa-ā-mua_ is still the 8th last (the 23rd).

**Comment by sttaseen** [comment]
You mentioned that the Tangaroa period varies between 3 to 4 nights depending on the number of nights in the lunar month. I am a bit confused about this. Most lunar months have about 29.5 days. I am a bit confused on how these nights are counted. So far, this is my reasoning:

If a full moon rises at 4:00 pm on a particular day, at night time (say, 6:00 pm approx.) on that day, this full moon should be available to be seen. So, we count this night even though the full moon rose before the night.  But then, what happens if the full moon rises at day time, like 11:00 am. Do we count the night before or do we count the night after or both?

**Reply by kumkee** [reply]
> You mentioned that the Tangaroa period varies between 3 to 4 nights depending on the number of nights in the lunar month. 

It turns out the length of  a Tangaroa period doesn't matter, but where it starts does.

> Most lunar months have about 29.5 days. How is it possible for there to be 30 nights? 
> Is there a numeric way to define nights?
 
The length of a lunar month is the number of calendar days, but not the exact time interval between the astronomical events.

**Reply by sttaseen** [reply]
So, does this mean we can assume that the Tangaroa period is always 4 days?

**Reply by kumkee** [reply]
> So, does this mean we can assume that the Tangaroa period is always 4 days?

It doesn't matter where it ends.

**Reply by sttaseen** [reply]
ok thank you!
